app/src/main/java/nextstep/shoppingcart/component/cart/CartTopBar.kt

```diff
@@ -0,0 +1,44 @@
+package nextstep.shoppingcart.component.cart
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowBack
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import nextstep.shoppingcart.R
+
+@Composable
+fun CartTopBar(
+    modifier: Modifier = Modifier,
+    onNavigationClick: () -> Unit,
```

Comment:
```suggestion
    onNavigationClick: () -> Unit,
    modifier: Modifier = Modifier,
```
Compose에서 파라미터의 순서는 "필수 > 옵셔널" 순서입니다.
Compose에서 권장하는 규칙을 지켜보면 어떨까요?

> Required parameters.
Single modifier: Modifier = Modifier.
Optional parameters.
(optional) trailing @Composable lambda.

ref. [Parameters order](https://android.googlesource.com/platform/frameworks/support/+/androidx-main/compose/docs/compose-component-api-guidelines.md#parameters-order)